ctp2_code/ui/aui_common/aui_pixel.cpp

```diff
@@ -111,12 +111,12 @@ AUI_ERRCODE aui_Pixel::Convert24To16Dither(
 	if (!thisrerr || !thisgerr || !thisberr ||
 		!nextrerr || !nextgerr || !nextberr)
     {
-		delete thisrerr;
-		delete thisgerr;
-		delete thisberr;
-		delete nextrerr;
-		delete nextgerr;
-		delete nextberr;
+		delete[] thisrerr;
```

Comment:
Ah... here is 'this' used. I did not see this code before. I still think that in general we should not use it.